stix2elevator/stix_stepper.py

```diff
@@ -208,6 +313,9 @@ def step_file(fn, validator_options, encoding="utf-8"):
 def main():
     stepper_arg_parser = _get_arg_parser()
     stepper_args = stepper_arg_parser.parse_args()
+    initialize_options()
+    set_option_value("missing_policy", "use-extensions")
+    set_option_value("custom_property_prefix", "elevator")
```

Comment:
You could provide the actual default value from the command-line arguments

```suggestion
    set_option_value("custom_property_prefix", stepper_args.custom_property_prefix)
```

Reply:
Dome